app/helpers/physical_switch_helper/textual_summary.rb

```diff
@@ -30,9 +30,16 @@ def textual_group_firmware_details
     TextualTable.new(_("Firmwares"), firmware_details, [_("Name"), _("Version")])
   end
 
+  def textual_group_connected_components
+    TextualGroup.new(
+      _("Connected Components"),
+      %i(connected_physical_servers)
+    )
+  end
+
   def textual_ports
     ports_count = @record.physical_network_ports.count
-    ports = {:label => _("Ports"), :value => ports_count, :icon => "ff ff-network-port"}
+    ports = {:label => _("Ports"), :value => ports_count, :icon => PhysicalNetworkPortDecorator.fonticon}
```

Comment:
I'd say `record.decorate.fonticon`, what do you think, @skateman ?

Reply:
yes

Reply:
In this case `@record` is a `PhysicalSwitch` instance, and here I'm printing the `PhysicalNetworkPort` icon, so if I put `record.decorate.fonticon` the wrong icon will be shown. :thinking:

Reply:
oh, ok. @skateman, if you have a better way to handle this, let's do it in a follow up PR.

Reply:
@martinpovolny :thinking: :thinking: :thinking: I don't have a better idea right now :confused: